pkg/kubernetes/schema/schema.json

```diff
@@ -1,6 +1,31 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "definitions": {
+    "config": {
```

Comment:
Currently the subschema that the mixins return in their schema command aligns pretty well with the structure in the porter.yaml. The mixin could decide to use their sub-schema to validate the input given to it when porter runs the mixin and passes related portions of the manifest.

I was expecting the config schema to do that as well and thought it would look like this:

mixins -> MIXIN -> schema for the mixin config

I don't think that any of the mixins rely on that but it seems nice to preserve for this as well unless it causes any problems. Does that sound doable?

Reply:
Would you able to sketch out what the envisioned corresponding json schema might look like?

Reply:
Sure! I think it is a small change but lets the mixin validate against the input we will send them https://github.com/getporter/kubernetes-mixin/commit/d162d141ef7dcf649d2bbf3556e88a4b968ca93c